languages/cy.po

```diff
@@ -19990,3 +19990,15 @@ msgstr "&copy; Hawlfraint"
 #: hale/partials/footer-licence.php
 msgid "All content is available under the <a class=\"govuk-footer__link\" href=\"%s\" rel=\"license\">Open Government Licence v3.0</a>, except where otherwise stated"
 msgstr "Mae’r holl gynnwys ar gael o dan <a class=\"govuk-footer__link\" href=\"%s\" rel=\"license\">Drwydded Llywodraeth Agored v3.0</a>, ac eithrio lle nodir yn wahanol"
+
```

Comment:
I think we need more translations.  These are the Welsh from the GDS PR:
```
      text: Gadael y dudalen
      activatedText: Tudalen ymadael
      timedOutText: Wedi’i amseru
      pressTwoMoreTimesText: Pwyswch 'Shift' 2 gwaith arall
      pressOneMoreTimeText: Pwyswch 'Shift' 1 mwy o amser
```
The last three texts are visually hidden texts for screen readers - they will be announced when shift keys are pressed.  

Also note how the translations put Shift in quotes, maybe that is a good idea.